bpf/lib/eps.h

```diff
@@ -98,7 +98,6 @@ ipcache_lookup4(struct bpf_elf_map *map, __be32 addr, __u32 prefix)
 	return map_lookup_elem(map, &key);
 }
 
-#if defined LXC_ID || defined SOCKMAP
```

Comment:
Please explain in the commit message why removal of the `#if` is OK

Reply:
From the new commit message:

```
    The wrappers around lookup_ip*_remote_endpoint() wrap the definition of
    macros to rewrite lookups into the ipcache map. In the non-LPM case,
    the IPCACHE_PREFIXES macro must be defined to ensure that the lookups
    find all ipcache entries; in the LPM case, so long as the macro is being
    used it must be defined. If it is defined and not used, the compiler
    should compile out the unnecessary code.
    
    The policy_can_egress*() functions only called from the egress BPF
    program which is attached to lxc devices, so the LXC_ID check there was
    a no-op.
    
    These changes should have no functional impact.
```